iptables/README.md

```diff
@@ -0,0 +1,65 @@
+iptables Extension
+==================
+
+This extensions adds four tables:
+
+* `iptables_ext`
+* `ip6tables_ext`
+* `iptables_policies`
+* `ip6tables_policies`
+
+## `iptables_ext` and `ip6tables_ext`
+
+### Schema
+
+| Column        | Type    | Description                                              |
+|:--------------|:--------|:---------------------------------------------------------|
+| filter_name   | TEXT    | Packet matching table name.                              |
```

Comment:
this issue exists with the core iptables table also, so i don't think anything should be done, but i think the name for this column should probably be "table_name" or something, right?

Reply:
Yep. I hadn't renamed it because it was low-pri, but now's the time to do it!